docs: Add a note about the compatibility between helm chart versions and Prometheus 3.0 (#5813)
SumoLogic/sumologic-documentation: docs/send-data/kubernetes/v4/important-changes.md

## Important changes

## Prometheus 3.0

> There are a number of breaking changes in Prometheus 3.0. Learn more about those changes and migration guide in the [documentation](https://prometheus.io/docs/prometheus/latest/migration/)
> 
> Please use Helm chart [v4.14.0](https://github.com/SumoLogic/sumologic-kubernetes-collection/releases/tag/v4.14.0) or later for compatibility with scrapers in Prometheus 3.0

Reviewer: ```suggestion
Prometheus 3.0 includes several breaking changes. Learn more about those changes and the migration guide in the [documentation](https://prometheus.io/docs/prometheus/latest/migration/).

Use Helm chart [v4.14.0](https://github.com/SumoLogic/sumologic-kubernetes-collection/releases/tag/v4.14.0) or later to ensure compatibility with Prometheus 3.0 scrapers.
```
